doc/source/beginner/overview.rst

```diff
@@ -69,14 +69,70 @@ qibuild is composed of two parts:
   taking dependencies into account and generate re-distributable binary
   packages.
 
+What makes qiBuild different ?
+++++++++++++++++++++++++++++++
 
-Is qibuild the only one build framework?
-++++++++++++++++++++++++++++++++++++++++
+The good parts
+~~~~~~~~~~~~~~
 
+* Full support for Visual Studio
 
-Of course not!
+* Full support for cross-compilation (hosts: linux, mac: targets : x86, arm, android, ...)
 
-You can have a loot at
+* Comes with a tool to use pre-compiled dependencies
+
+* No environment variables required, and keep your environment clean
+
+* Written in Python like many others but:
+
+  * Python2/Python3 compatible
+
+  * >80% test coverage
+
+* Can find dependencies **from the sources**. For instance, in a worktree with
+  two different CMake projects , ``world`` and ``hello``, when compiling ``hello``,
+  we will find ``world`` headers directly from ``world`` sources.
```

Comment:
maybe add "and world library from build directory, without requiring an install step."